quipucords/fingerprinter/runner.py

```diff
@@ -82,6 +82,42 @@
 COMBINED_KEY = "combined_fingerprints"
 
 
+def fingerprint_network_infrastructure_type(fact: dict) -> tuple[str, str]:
+    """Determine if running on VM or bare metal."""
+    virt_what_type = fact.get("virt_what_type")
+    hostnamectl_chassis = (
+        (fact.get("hostnamectl", {}) or {}).get("value", {}).get("chassis")
+    )
```

Comment:
Ah, I just remembered now we have a utility for this

```suggestion
    hostnamectl_chassis = deepget(fact, "hostnamectl__value__chassis")
```